client/src/components/AlternateApp.jsx

```diff
@@ -70,8 +70,12 @@ export default class AlternateApp extends Component {
     cohortsQuery()
       .then(result => {
         const allCohorts = result.data.data.cohorts;
-        const sprintCohorts = allCohorts.filter(cohort => cohort.phase === 'sprint');
-        const teamCohorts = allCohorts.filter(cohort => cohort.phase === 'project');
+        const sprintCohorts = allCohorts.filter(
+          cohort => cohort.phase === 'sprint' && cohort.status === 'current'
```

Comment:
Let's iterate on this to make it a little easier as a next step.  Let's put a cohort start and a cohort graduation date -- maybe even a TA date -- and then do a date check to ensure today's date falls between both.  Then we can put several years worth of cohort data in config without having to maintain "current" status.

Reply:
Or we can add a cron job once a week to check the dates and alter the status automatically.